IDE/Renesas/e2studio/RX72N/EnvisionKit/wolfssl_demo/wolfssl_demo.h

```diff
@@ -59,5 +57,28 @@
 /* cannot enable with other definition */
 #define TLS_CLIENT
 
+/* use multi-thread example */
+/* #define TLS_MULTITHREAD_TEST */
+#if defined(TLS_MULTITHREAD_TEST) && defined(WOLFSSL_TLS13)
+    #error "MULTITHREAD_TEST is only available when not set WOLFSSL_TLS13"
```

Comment:
I expected to have it worked. However, there is error while using TLS 1.3 suite under multi threading. It appears that new TSIP TLS1.3 support has issue under multi threads. Because it would take time to fix, I simply disallow it for now.

How about "MULTITHREAD_TEST is only available when not set WOLFSSL_TLS13 because it is not yet verified" as message?